crt_portal/cts_forms/models.py

```diff
@@ -226,6 +226,16 @@ def get_absolute_url(self):
         return f'/form/view?{self.query}'
 
 
+class RetentionSchedule(models.Model):
+    name = models.CharField(max_length=255, null=False, blank=False, help_text="The name of the schedule that will be shown to intake specialists in dropdowns.")
+    order = models.IntegerField(default=0, help_text="The order in which to show the schedules, lower numbers first. If two schedules have the same order number, they might change positions.")
+    description = models.TextField(max_length=7000, null=False, blank=True, help_text="Internal notes, shown only here, about this schedule.")
+    tooltip = models.CharField(max_length=255, null=False, blank=True, help_text="The text to show in the tooltip for this schedule.")
+    retention_years = models.IntegerField(null=False, blank=False, help_text="The number of years to retain reports with this schedule (following their closure). Set to 0 for permanent retention.")
+    da_number = models.CharField(max_length=255, null=False, blank=False, help_text="The disposition authority number for this schedule (assigned by NARA).")
+    is_retired = models.BooleanField(default=False, null=False, help_text="Whether this schedule is no longer active (show as a retired / expired schedule)")
```

Comment:
Could you clarify what use case this is for? Is this for when the retention period expires?

Reply:
Yeah - when the schedule itself is replaced by another schedule and should no longer be assigned to new reports.